app/assets/javascripts/discourse/app/templates/user-selector-autocomplete.hbr

```diff
@@ -6,7 +6,11 @@
           {{avatar user imageSize="tiny"}}
           <span class='username'>{{format-username user.username}}</span>
           {{#if user.status}}
-            {{emoji user.status.emoji title=user.status.description}}
+            {{emoji user.status.emoji}}
+            <span class='status-description'>{{user.status.description}}</span>
```

Comment:
Is there a limit on the length of the description? I'm wondering how the UI looks when the description is too long.

Reply:
We don't have a limit on the status description now, but we plan to add it. Anyway, limit will be longer than it has to be to fit into free space on this popup.

Here is how a truncated status looks like in this place:
<img width="332" alt="Screenshot 2022-08-29 at 19 26 04" src="https://user-images.githubusercontent.com/1274517/187237382-76551245-7a3d-4055-81d9-32ac2e440b02.png">

A little problem here is that, we don't show endsAt time in this case. Seems like a minor thing to me, but I'm leaving a note to myself to fix it.

Reply:
Hmm personally I don’t think this is minor because we’re assuming descriptions that are short when in reality a few words will easily cause the description to be truncated. I’m wondering if we minimally need a title attribute on the element here?

Reply:
Oh, right! A title attribute is a must here, I've added it:

<img width="534" alt="Screenshot 2022-08-29 at 19 44 44" src="https://user-images.githubusercontent.com/1274517/187241100-6e3c5190-d53d-486f-8964-443636418a40.png">